iOS/Collection View Management /DiffableCollectionViewController.swift

```diff
@@ -156,7 +156,8 @@ class DiffableCollectionViewController<SectionType: Hashable,
         return nil
     }
     
-    func collectionView(_ collectionView: UICollectionView, willDisplay cell: UICollectionViewCell, forItemAt indexPath: IndexPath) {
-        
-    }
+    func collectionView(_ collectionView: UICollectionView, willDisplay cell: UICollectionViewCell, forItemAt indexPath: IndexPath) {}
+    
+    func scrollViewWillBeginDragging(_ scrollView: UIScrollView) {}
+    func scrollViewDidEndDragging(_ scrollView: UIScrollView, willDecelerate decelerate: Bool) {}
```

Comment:
We should probably still call super in the override even though the function body is empty, just in case we add something here later.